src/critics/reference_trajectory_critic.cpp

```diff
@@ -13,48 +13,36 @@ void ReferenceTrajectoryCritic::initialize()
   getParam(reference_cost_power_, "reference_cost_power", 1);
   getParam(reference_cost_weight_, "reference_cost_weight", 3.0);
 
-  getParam(enable_nearest_path_angle_critic_, "enable_nearest_path_angle_critic", true);
-  getParam(nearest_path_angle_offset_, "nearest_path_angle_offset", 6);
-  getParam(nearest_path_angle_cost_power_, "nearest_path_angle_cost_power", 1);
-  getParam(nearest_path_angle_cost_weight_, "nearest_path_angle_cost_weight", 1.0);
-
-  getParam(enable_nearest_goal_critic_, "enable_nearest_goal_critic", true);
-  getParam(nearest_goal_offset_, "nearest_goal_offset", 2);
-  getParam(nearest_goal_count_, "nearest_goal_count", 2);
-  getParam(nearest_goal_cost_power_, "nearest_goal_cost_power", 1);
-  getParam(nearest_goal_cost_weight_, "nearset_goal_cost_weight", 1.0);
+  getParam(reference_point_step_, "reference_point_step", 1);
```

Comment:
These updates of new parameters / removing old should be updated in the readme table

Reply:
Might I suggest `plan_point_step` versus `reference`? Though I don't care that much

Reply:
renamed